server/follow/test/follow.test.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-env node, mocha */
+'use strict'
+
+const followPlugin = require('../index')
+
+const assert = require('assert')
+const nock = require('nock')
+const redis = require('redis')
+const client = redis.createClient()
+const Fastify = require('fastify')
+const fp = require('fastify-plugin')
+
+const makeRequest = (fastify, options) => new Promise((resolve) => fastify.inject(options, resolve))
```

Comment:
can you use `util.promisify` for this? Maybe we can even add Promise support in inject.

Reply:
I'll fire a PR for this!